ParseStarterProject/IntroViewController.swift

```diff
@@ -0,0 +1,119 @@
+/**
+* Copyright (c) 2015-present, Parse, LLC.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree. An additional grant
+* of patent rights can be found in the PATENTS file in the same directory.
+*/
+
+import UIKit
+import Parse
+import ParseUI
+
+class IntroViewController: UIViewController, PFLogInViewControllerDelegate, PFSignUpViewControllerDelegate {
+    
+    // Instantiate our login and sign up views
+    var loginViewController: PFLogInViewController = PFLogInViewController()
+    var signUpViewController: PFSignUpViewController = PFSignUpViewController()
+    
+    /*
+    * When the view appears, we need to check if the user is already logged in. If they are let's skip this screen
+    * and go straight to the main app.
+    */
+    override func viewDidAppear(animated: Bool) {
```

Comment:
The PFUser.currentUser() line? Yeah it's parse's way of returning the user from the current session if there is one